central/values.yaml

```diff
@@ -1413,6 +1413,7 @@ centralledger:
         wsrep_cluster_name=galera
         wsrep_sst_method=xtrabackup-v2
         pxc_strict_mode=DISABLED
+        sql_mode=STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
```

Comment:
1. strictly speaking, they're not being added, they are in fact defaults: https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html#sql-mode-setting

2. You're right, I'm surprised that, unless I'm misreading this, they appear to be both defaults _and_ deprecated. Fun times. I suppose there's probably some backwards compat justification for that.

_Anyway_. This has arisen because the finance portal has some queries that (it turns out) aren't semantically correct, but previously mysql has been more forgiving (even in some 5.7 < 5.7.11 versions). In fact, another solution to this issue is to identify the affected queries and make them semantically correct. I think probably we shouldn't make the proposed change to the sql mode, and instead we should modify the queries in question.

Reply:
Addressed by https://github.com/mojaloop/finance-portal-backend-service/pull/10

Reply:
And https://github.com/mojaloop/finance-portal-backend-service/issues/11